AzureMonitor/Scripts/Create-AzDiagPolicy.ps1

```diff
@@ -437,7 +450,7 @@ function Get-ResourceType (
                 catch
                 {
                     # Uncomment below to see actual error.  Certain resources are not ResourceTypes that can support Logs and Metrics so the host error is being muted
-                    #write-host $Error[0] -ForegroundColor Red
+                    # write-host $Error[0] -ForegroundColor Red
```

Comment:
Can we remove the space? That way someone removes the comment and proper indenting occurs. No need to add the space.

Reply:
Done in 
3d3d363